src/common/colorspaces.c

```diff
@@ -641,17 +648,98 @@ static cmsHPROFILE dt_colorspaces_create_linear_rec2020_rgb_profile(void)
   return profile;
 }
 
+#if 0
+static cmsHPROFILE dt_colorspaces_create_linear_prophoto_rgb_profile(void)
+{
+  // code taken from here:
+  // https://github.com/ellelstone/elles_icc_profiles/blob/master/code/make-elles-profiles.c
+  cmsCIExyY d50_romm_spec = {0.3457, 0.3585, 1.0};
+//  cmsCIEXYZ black = { 0, 0, 0 };
+
+  cmsCIExyYTRIPLE romm_primaries = {
+    {0.7347, 0.2653, 1.0},
+    {0.1596, 0.8404, 1.0},
+    {0.0366, 0.0001, 1.0}
+  };
+
+  cmsToneCurve *curve[3], *tonecurve;
+  tonecurve = build_linear_gamma();//cmsBuildGamma (NULL, 1.00);
+  curve[0] = curve[1] = curve[2] = tonecurve;
+
+  /* Make V4 profile */
+  cmsHPROFILE profile = cmsCreateRGBProfile ( &d50_romm_spec, &romm_primaries, curve);
+
+//  cmsWriteTag(profile, cmsSigMediaWhitePointTag, &d65);
+//  cmsWriteTag(profile, cmsSigMediaBlackPointTag, &black);
+
+  return profile;
+}
 static cmsHPROFILE dt_colorspaces_create_linear_prophoto_rgb_profile(void)
 {
   cmsToneCurve *transferFunction = build_linear_gamma();
 
   cmsHPROFILE profile = _create_lcms_profile("Linear prophoto RGB", "Linear prophoto RGB",
-                                             &prophoto_primaries_prequantized, transferFunction, TRUE);
+                                 &prophoto_primaries_prequantized, transferFunction, TRUE);
 
   cmsFreeToneCurve(transferFunction);
 
   return profile;
 }
+#else
+static cmsHPROFILE dt_colorspaces_create_linear_prophoto_rgb_profile(void)
+{
+  cmsCIEXYZ black = { 0, 0, 0 };
+
+  cmsContext icc_context;
+  cmsHPROFILE out_profile;
+  cmsMLU *mlu1, *mlu2;
+  cmsToneCurve *gamma18, *out_curves[3];
+//  uint32_t icclen;
+//  uint8_t *icc;
+  // http://www.brucelindbloom.com/index.html?WorkingSpaceInfo.html
+  static cmsCIExyYTRIPLE primaries = {
+    /*     x,      y,        Y */
+    { 0.7347, 0.2653, 0.288040 }, /* red   */
+    { 0.1596, 0.8404, 0.711874 }, /* green */
+    { 0.0366, 0.0001, 0.000086 }, /* blue  */
+  };
+
+  mlu1 = cmsMLUalloc(NULL, 1);
+  mlu2 = cmsMLUalloc(NULL, 1);
+
+  /* icc profile (ProPhoto RGB) */
+  icc_context = cmsCreateContext(NULL, NULL);
+  gamma18 = cmsBuildGamma(icc_context, 1.8);
```

Comment:
this is not a linear RGB space though

Reply:
Right, this was WIP. The good news is that I have all profile re-implemented and the pink cast is gone. I'll publish a PR later today.